sentinel/src/schedule/reminders.js

```diff
@@ -1,225 +1,295 @@
-    isConfigValid: isConfigValid,
-    getSchedule: getSchedule,
-    // called from schedule/index.js on the hour, for now
-    execute: callback => {
-        var reminders = config.get('reminders') || [];
-
-        async.eachSeries(reminders, function(reminder, callback) {
-            if (!isConfigValid(reminder)) {
-                return callback();
-            }
-            module.exports.runReminder({
-                reminder: reminder
-            }, callback);
-        }, callback);
-    },
-    // matches from "now" to the start of the last hour
-    // later reverses time ranges fro later#prev searches
-    matchReminder: function(options, callback) {
-        var start = moment(),
-            reminder = options.reminder,
-            sched = getSchedule(reminder);
-
-        // this will return a moment sometime between the start of the hour and 24 hours ago
-        // this is purely for efficiency so we're not always examining a 24 hour stretch
-        module.exports.getReminderWindow(options, function(err, end) {
-            // this will return either the previous time the schedule schould have run
-            // or null if it should not have run in that window.
-            var previous = sched.prev(1, start.toDate(), end.toDate());
-
-            if (_.isDate(previous)) {
-                callback(null, moment(previous));
-            } else {
-                callback(null, false);
-            }
-        });
-    },
-    canSend: function(options, clinic) {
-        var send,
-            ts = options.moment || moment().startOf('hour'),
-            reminder = options.reminder,
-            lastReceived,
-            muteDuration;
-
-        // if it's already been sent, it will have a matching task with the right form/ts
-        send = !_.findWhere(clinic.tasks, {
-            form: reminder.form,
-            timestamp: ts.toISOString()
-        });
-
-        // if send, check for mute on reminder, and clinic has sent_forms for the reminder
-        // sent_forms is maintained by the update_sent_forms transition
-        if (send && reminder.mute_after_form_for && clinic.sent_forms && clinic.sent_forms[reminder.form]) {
-            lastReceived = moment(clinic.sent_forms[reminder.form]);
-            muteDuration = module.exports.parseDuration(reminder.mute_after_form_for);
-
-            if (lastReceived && muteDuration) {
-                // if it should mute due to being in the mute duration
-                send = ts.isAfter(lastReceived.add(muteDuration));
-            }
-        }
+// matches from "now" to the start of the last hour
+// later reverses time ranges fro later#prev searches
+const matchReminder = (reminder) => {
+  const schedule = getSchedule(reminder);
+  // this will return a moment sometime between the start of the hour and 24 hours ago
+  // this is purely for efficiency so we're not always examining a 24 hour stretch
+  return getReminderWindow(reminder)
+    .then(end => {
```

Comment:
yea, I guess it was in  conjunction with that comment `later reverses time ranges fro later#prev searches` that I removed. 
I changed it to respect the time axis direction instead of the `later.prev` param order.